script/Arcueid/Day 7/07_00_ARC07_1C.txt

```diff
@@ -198,7 +198,7 @@ Arihiko shows me the screen of his phone.
 [sha:1799df0100f04341a147f887f1778058ca41cfb2]{
 -- Page 105, Offset 16828. Glued. Mods: @x.
 -- え」
-Eh?"
+Huh?"
```

Comment:
Drop the comma on the em-dash before this huh

Reply:
already did in the previous pr